src/sql/magic.py

```diff
@@ -401,6 +406,9 @@ def interactive_execute_wrapper(**kwargs):
                 "is automatically set as the connection alias"
             )
 
+        # TODO: DuckDB allows queries to be wrapped in parentheses
+        # which needs to be accounted for here. If there is a (WITH ....)
+        # query, then there is an issue of is_CTE not being true.
```

Comment:
remove this. if we already have an open issue, there's no need for this note

Reply:
Will do!

Reply:
Removed the comments